### DISCUSSION TITLE
Hey is CodeSandbox just dead now?

### DISCUSSION THREAD

**Discussion by corysimmons** [codesandbox/codesandbox-client#8252] [General]
Every single sandbox I go to is just this:

![image](https://github.com/codesandbox/codesandbox-client/assets/224636/7723e60c-17d4-4f64-8426-b1814bd56f48)

I noticed uBlock Origin is blacklisting the domain now, so I turned off all my adblock stuff hoping that was it, but it didn't seem to help. I tried in an Incognito window with no extensions and it didn't help. Tried in Firefox with no extensions and it didn't help.

I finally got curious and made a fresh sandbox with Next.js and it was slllloowww (like an insane amount), then noticed you're charging a subscription for fast boxes.

**What is going on?** I genuinely don't understand if this is just some upgrade process where you accidentally broke the site for weeks (months?) now, or if you're just trying to cash in. Whatever the reason, I can't access demos for any of the **countless** open-source projects that trusted CodeSandbox.

Not trying to be rude. I love you guys and just really frustrated all the doc demos I try to access are broken.

**Comment by JamesACS** [comment]
Hello Cory, 

Please could you contact us directly at support@codesandbox.io? 

We would need specific links to identify issues in any environments and haven't had any reports of widespread malfunctions on the platform, so we'll need to run through some troubleshooting steps.

**Reply by corysimmons** [reply]
Sure thing. Let me start collecting them and I'll send you something.
